indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/supervisor/SeekableStreamSupervisor.java

```diff
@@ -1640,19 +1648,31 @@ private boolean isTaskCurrent(int taskGroupId, String taskId)
     SeekableStreamIndexTask<PartitionIdType, SequenceOffsetType> task = (SeekableStreamIndexTask<PartitionIdType, SequenceOffsetType>) taskOptional
         .get();
 
-    String taskSequenceName = task.getIOConfig().getBaseSequenceName();
```

Comment:
Every task in a task group needs to have the same sequence name (so replicas segment allocations match up), meaning you _do_ need to remember `task.getIOConfig().getBaseSequenceName()`, and should probably assign it as the sequence name for the relevant task group before starting to create new tasks.